web/src/scripts/utils/ElementTreeUtils.js

```diff
@@ -59,13 +59,28 @@ export const deepUpdateProp = (tree, elementPath, propName, value, text) => {
   const element = getElementByPath(newTree, elementPath)
 
   const {props = []} = element
-  const prop = props.find(x => x.name === propName)
-
-  const oldLength = prop.valueEnd.ch - prop.valueStart.ch
-  const newLength = text.length
+  let objectProp = null
```

Comment:
I don't really follow the intention here... maybe can be written more clearly, or if not it needs comments